torchvision/datasets/__init__.py

```diff
@@ -24,6 +24,7 @@
 from .hmdb51 import HMDB51
 from .ucf101 import UCF101
 from .places365 import Places365
+from .kitti import Kitti
```

Comment:
I'd even suggest to name new files with a leading underscore, so that all objects defined in this file are private by default. But I'd confirm with @fmassa first before making this change.

```suggestion
from ._kitti import Kitti
```

Reply:
This is a good point. I think for the sake of consistency we should just keep `kitti` as is for now, but we can discuss this in an issue.